manager/integration/tests/test_upgrade.py

```diff
@@ -75,6 +80,24 @@ def get_longhorn_upgrade_type():
     return [os.environ.get('LONGHORN_UPGRADE_TYPE', '')]
 
 
+def create_volume_and_wrte_date(client, volume_name):  # NOQA
```

Comment:
We should have this kind of method already in common.py, isn't it?

Reply:
> We should have this kind of method already in common.py, isn't it?

It's due to https://github.com/longhorn/longhorn-tests/pull/1168#discussion_r1032009595.  
We have `create_volume` & `write_volume_random_data`, but we don't have those in the same function in common.py.

Reply:
Hi @khushboo-rancher 
Fix the typo and move create_volume_and_write_data to common.py
  - https://ci.longhorn.io/job/private/job/longhorn-tests-regression/2897/
  
![Screenshot_20230110_161817](https://user-images.githubusercontent.com/104337648/211497952-110d6a23-9768-4505-b04e-c3fe50c0c1bc.png)